ci/test_integration.sh

```diff
@@ -20,6 +20,8 @@ echo "==== ASR (backend=pytorch, lm=TransformerLM) ==="
 ./run.sh --python "${python}" --stage 5 --lm-config conf/lm_transformer.yaml --decode-config "$(change_yaml.py conf/decode.yaml -a api=v2)"
 echo "==== ASR (backend=pytorch, dtype=float64) ==="
 ./run.sh --python "${python}" --stage 3 --train-config "$(change_yaml.py conf/train.yaml -a train-dtype=float64)" --decode-config "$(change_yaml.py conf/decode.yaml -a api=v2 -a dtype=float64)"
+echo "==== ASR (backend=pytorch, quantize-asr-model true, quantize-lm-model true) ==="
+./run.sh --python "${python}" --stage 5 --decode-config "$(change_yaml.py conf/decode.yaml -a quantize-asr-model=true -a quantize-lm-model=true)"
```

Comment:
This line failed because some CI tests old pytorch. @kamo-naoyuki is there any common way to detect and skip old pytorch in shell script?

Reply:
I think we can use `matrix.pytorch-version` variable in ci.yml https://github.com/espnet/espnet/blob/ba564fc023765e1e0d6ebcb04efc1cb0da7b6205/.github/workflows/ci.yaml#L19
For example,
```yml
# ci.yml
- name: test integration
  env:
    TEST_QUANTIZATION: ${{ !contains(["1.0.1", "1.1.0", "1.2.0"], matrix.pytorch-version) }} 
  run: ./ci/test_integration.sh
```
```bash
# ci/test_integration.sh
if [ ${TEST_QUANTIZATION:-1} -eq 1 ]; then
  echo "==== ASR (backend=pytorch, quantize-asr-model true, quantize-lm-model true) ==="
  ./run.sh --python "${python}" --stage 5 --decode-config "$(change_yaml.py conf/decode.yaml -a quantize-asr-model=true -a quantize-lm-model=true)" 
fi
```

Reply:
thx

Reply:
thx

Reply:
My suggest was wrong. The string quote must be single '  instead of "
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#literals